addons/armaos/CfgVehicles.hpp

```diff
@@ -167,8 +167,8 @@ class CfgVehicles
 					class AE3_UseComputer
 					{
 						displayName = "Use";
-						condition = "(alive _target) and (_target getVariable 'AE3_power_powerState' == 1)";
-						statement = "params ['_target', '_player', '_params']; _handle = [_target] call AE3_armaos_fnc_terminal_init;";
+						condition = "(alive _target) && (_target getVariable 'AE3_power_powerState' == 1) && (isNull (_target getVariable ['AE3_computer_mutex', objNull]))";
```

Comment:
If this is kind of a locking mechanism, I think it would be better to additionally check if player == AE3_computer_mutex, especially in cases where the player stops using armaOS not by triggering shutdown or standby, for example a disconnect or some weird interface thingy that allows you to close the dialog in a bypass kind of way. In this case the player could return to his own "session". What do you think?

Reply:
The lock is removed, if the dialog is closed. This should in theory also cover cases, when the dialog is
closed unintentionally. 
Because the player object is referenced in the variable, it should also become null, if the player
disconnects from the server. So the lock is also lifted in this case.

Reply:
https://github.com/y0014984/Advanced-Equipment/blob/674583d280f7f4a7310168659b1a17d0b9445edd/addons/armaos/functions/fnc_terminal_addEventHandler.sqf#L204

Reply:
I saw you already managed this event by adding the Display Unload Event Handler. I think this would be sufficient.